css/base/layout.css

```diff
@@ -242,7 +242,7 @@
         display: none !important;
     }
     .container > .row {
-        width: 720px;
+        width: 750px;
```

Comment:
dėl kokios priežasties pakeistas šis dydis?

Reply:
Nepamenu ką tiksliai tikrinau, bet jo neprireikė. Tai tuoj atkeisiu atgal